arches/app/etl_modules/bulk_data_deletion.py

```diff
@@ -129,19 +130,34 @@ def get_sample_data(self, nodegroup_id, resourceids):
 
         return sample_data[0:5]
 
-    def delete_resources(self, userid, loadid, graphid, resourceids):
+    def delete_resources(self, userid, loadid, graphid, resourceids, verbose=False):
         result = {"success": False}
-        user = User.objects.get(id=userid)
+        deleted_count = 0
+        try:
+            user = User.objects.get(id=userid)
+        except ValueError:
+            user = None
```

Comment:
I would have expected User.DoesNotExist, not ValueError. Did you see this during development?

Reply:
if it's passed `None` it will raise a `ValueError`, otherwise if wold raise `DoesNotExist`